.prettierrc

```diff
@@ -1,5 +1,5 @@
 {
   "printWidth": 100,
   "singleQuote": true,
-  "trailingComma": "es5"
+  "trailingComma": "All"
```

Comment:
Is it `"All"` or `"all"`?

Reply:
It shud be "all", typo error, will correct that